airflow_connector.py

```diff
@@ -0,0 +1,70 @@
+"""Connects to the Cloud SQL database."""
+import os
+import MySQLdb
+
+# These environment variables are configured in app.yaml.
+CLOUDSQL_CONNECTION_NAME = os.environ.get('CLOUDSQL_CONNECTION_NAME')
+CLOUDSQL_USER = os.environ.get('CLOUDSQL_USER')
+CLOUDSQL_PASSWORD = os.environ.get('CLOUDSQL_PASSWORD')
+CLOUDSQL_HOST = os.environ.get('CLOUDSQL_HOST')
+CLOUDSQL_DB = os.environ.get('CLOUDSQL_DB')
+
+if CLOUDSQL_CONNECTION_NAME is None:
+  CLOUDSQL_CONNECTION_NAME = 'istio-release-ui:us-central1:prod-airflow-snapshot-sandbox'  # pylint: disable=line-too-long
+  CLOUDSQL_USER = 'root'
+  CLOUDSQL_PASSWORD = ''
+  CLOUDSQL_HOST = '35.193.234.53'
+  CLOUDSQL_DB = 'airflow-db'
+
+
+class AirflowDB(object):
+  """"Provides the methods which allow interaction with the Airflow SQL database."""  # pylint: disable=line-too-long
+
+  def __init__(self):
```

Comment:
you should be passing everything as arguments in init not thru global variable. When you instantiate AirflowDB you can pass them via flags in your main.